src/core/cgroup.c

```diff
@@ -1580,7 +1581,7 @@ void unit_modify_nft_set(Unit *u, bool add) {
 
         assert(u);
 
-        if (!MANAGER_IS_SYSTEM(u->manager))
+        if (!manager_is_system(u->manager))
```

Comment:
Please keep these upper case. It's fine for macro-like functions too

Reply:
We should probably have something in CODING_STYLE.md about this. I don't really see the point in keeping these upper case if they're not macros

Reply:
Actually I'm not sure this is needed anymore with the latest approach, will check and remove these changes then.

Reply:
Dropped